inc/class-qm-collector-notifications.php

```diff
@@ -0,0 +1,35 @@
+<?php
+/**
+ * Data collector class
+ */
+
+namespace Altis\Workflow;
+
+use QM_Collector;
+
+class QM_Collector_Notifications extends QM_Collector {
+
+	public $id = 'workflow_notifications';
+
+	/**
+	 * Defines the name of the collector.
+	 *
+	 * @return string|void
+	 */
+	public function name() {
```

Comment:
I _think_ this method has been deprecated in this class and should be moved to the output HTML class, check your debug log to see if you're getting this notice as I remember seeing it on my local.

Reply:
no I don't get any notices about that